domains/cards/services.ts

```diff
@@ -70,12 +70,19 @@ export function updateCard(id: string, updatedCard: Card) {
 
 export function addRecordToCard(cardId: string, record: CardRecord) {
   const card = getCardById(cardId)
+  if (!card) {
+    return
+  }
   card.records.push(record)
+  card.records.sort((r1, r2) => (r1.age < r2.age) ?-1 : (r1.age > r2.age) ? 1 : 0);
```

Comment:
It's not required to return `-1`, `0`, `1` from `sort` callback. You can return any number, and sort order will depend on the sign of the returned number. [See more here (Numeric sort)](https://www.w3schools.com/js/js_array_sort.asp)

```suggestion
  card.records.sort((r1, r2) => r1.age - r2.age);
```